### PR TITLE
Frontend and related workflows

### PR DESCRIPTION
**Pull request by DerekRoberts**
---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-14-backend.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://nr-spar-14-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -125,19 +122,44 @@ jobs:
...
+    if: github.event_name != 'pull_request' || !github.event.pull_request.draft
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: bcgov-nr/action-test-and-analyse@v0.0.1
+        with:
+          commands: |
+            yarn --prefer-offline
+            yarn run lint
+            yarn build
+            yarn test -u --coverage
```
Comment (RMCampos) [inline .github/workflows/ci.yml:142]: I think these commands should include directory location, like `yarn --cwd frontend --prefer-offline`, `yarn --cwd frontend run lint` and so on

**Review by RMCampos — DISMISSED** [review]
One change about frontend location and should be good to go.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (DerekRoberts) [reply .github/workflows/ci.yml:142]: Almost!  `dir: frontend` took care of that.  ;)

Comment (DerekRoberts) [reply .github/workflows/ci.yml]: Update: @RMCampos was right.  Whoops!

**Pull request merged**